backend/typescript/types.ts

```diff
@@ -85,7 +90,7 @@ export type CamperCSVInfoDTO = {
   "Requires Early Drop-off": string;
   "Requires Late Pick-up": string;
   // eslint-disable-next-line
-  "Allergies": string;
+  Allergies: string;
```

Comment:
Why are we making this change?

Reply:
This was an automatic linter change for me